src/test/java/com/wikia/webdriver/common/logging/VelocityWrapper.java

```diff
@@ -0,0 +1,179 @@
+package com.wikia.webdriver.common.logging;
+
+import org.apache.velocity.Template;
+import org.apache.velocity.VelocityContext;
+import org.apache.velocity.app.VelocityEngine;
+
+import java.io.StringWriter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * Created by szymonczerwinski on 01.03.2017.
+ */
+public class VelocityWrapper {
+
+  StringBuilder builder;
+  VelocityEngine ve;
+
+  public VelocityWrapper(){
+    builder = new StringBuilder();
+    ve = new VelocityEngine();
+    ve.init();
+  }
+  public static String appendLogRow(List<String> classList, String command, String description) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRow.vm");
+    VelocityContext context = new VelocityContext();
+
+    context.put("className", "\"" + classList.stream().collect(Collectors.joining(" ")) + "\"");
+    context.put("command", command);
+    context.put("escapedDescription", description);
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
+  }
+
+  public static String appendLastLogRow() {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/lastLogRow.vm");
+    VelocityContext context = new VelocityContext();
+
+    StringWriter writer = new StringWriter();
+    t.merge(context, writer);
+    builder.append(writer.toString());
+    return builder.toString();
+//    CommonUtils.appendTextToFile(logPath, builder.toString());
+  }
+
+  public static String appendLogRowWithLink(String link, String label) {
+    VelocityEngine ve = new VelocityEngine();
+    StringBuilder builder = new StringBuilder();
+
+    Template t =
+        ve.getTemplate("./src/test/java/com/wikia/webdriver/common/velocitytemplates/logRowWithLink.vm");
+    VelocityContext context = new VelocityContext();
+    context.put("link", link);
+    context.put("label", label);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "label" 3 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)